modalsheet/src/main/java/eu/wewox/modalsheet/ModalSheet.kt

```diff
@@ -146,8 +150,10 @@ public fun ModalSheet(
 
     LaunchedEffect(visible) {
         if (visible) {
+            internalCancelable.value = cancelable
```

Comment:
this launched effect is not re-run when `cancelable` changes but is the only way how `cancelable` can propagate into `internalCancelable`. I don't really know much about this domain to point to a bug right away but looks very error prone

Reply:
Thanks for a review 🫶  

Yeah, there could be use-cases where `cancelable` is updated when the sheet is shown, and in this case, it will not work correctly.

I have added `cancelable` to the effect keys. It seems to have fixed the issue with updating the flag. As a small side effect, the `show()` function will be called once more, but that's OK as for me, as it only animates the sheet to the expanded state (which is already expanded).

Additionally, I've re-added the "update parameters" logic to update the back handler as well.